06week/checkpoint2.js

```diff
@@ -0,0 +1,127 @@
+
+
+// HIGHER ORDER FUNCTIONS USED
+//MAP
+// map method creates a new array with the results of calling a provided 
+//function on every element in the calling array.
+//FILTER
+// creates a new array with all elements that pass the test implemented by the provided function.
+//REDUCE
+// method applies a function against an accumulator and each element in the array (from left to right) to reduce it to a single value.
+// FIND 
+// returns the value of the first element in the array that satisfies the provided testing function. Otherwise undefined is returned.
+// FINDINDEX
+//The findIndex() method returns the index of the first element in the array that satisfies the provided testing function. Otherwise -1 is returned.
+ 
+
+
+
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+ const nums = strNums.map(numbers => Number(numbers));
+// builds new array that takes the numbers in the strNums array and 
+// rolls out the the function over each array 
+// function = string data type into number datatype
+
+
+const sumEvens = numbers.filter(number => {
```

Comment:
numbers isn't a defined array that can be filtered.  The variable that was defined was nums